### DISCUSSION TITLE
Add the heatmap alone in the visualizer `mode="full"`

### DISCUSSION THREAD

**Discussion by jpcbertoldo** [open-edge-platform/anomalib#683] [Feature Requests]
In the visualizer in mode full we have the image and the image with the heatmap superposed. 
I'd like to also see the heatmap alone, without superposing it to the image.

[Here](https://github.com/openvinotoolkit/anomalib/blob/3de3eefc58aee6538a7b2a8b23cf62d287164b1b/anomalib/post_processing/visualizer.py#L122):

![image](https://user-images.githubusercontent.com/24547377/200013494-55247ebc-9a12-430f-85c9-68fd9514aebc.png)

It's a 5-ish line PR : )
I did it on my side.

What do you think?

**Comment by samet-akcay** [comment]
@jpcbertoldo, tidying up the discussions, is this still relevant?
